src/hooks/useFarmStats.js

```diff
@@ -1,33 +1,39 @@
 import { useEffect, useState } from 'react'
 
+import JSBI from 'jsbi'
```

Comment:
No need to use JSBI—[Modern browsers already support](https://caniuse.com/bigint) native `BigInt` (even Safari)